oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/DocumentPropertyStateTest.java

```diff
@@ -16,14 +16,17 @@
  */
 package org.apache.jackrabbit.oak.plugins.document;
 
-import java.io.IOException;
-import java.io.InputStream;
+import java.io.*;
```

Comment:
Please avoid `*` imports and import only required packages.

Reply:
Done